src/components/Resource/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./index.css";
+import { firebaseStorage } from "../../firebaseConfig";
+
+const handleDownloadFileFromFirebaseStorage = async (resource) => {
+  try {
+    let url = await firebaseStorage.storage().refFromURL(resource).getDownloadURL();
+    window.open(url);
+  } catch (e) {
+    console.error(e);
+    alert(`Couldn't get the download link`);
+  }
+};
+const Resource = ({ resource }) => {
+  if (!resource) {
+    return (<h2>Loading</h2>)
+  }
+  else {
+  return (
+    <div className="resource__body">
+      <div className="resource__body__picContainer">
+        <img
+          src={resource.pic}
+          alt={resource.title}
+          className=""
```

Comment:
empty

Reply:
I think it's added by mistake. since there is a className already.